setup.py

```diff
@@ -61,15 +62,28 @@ def run(self):
     # Build regular proto files.
     protos = glob.glob(os.path.join(self.indir, '*.proto'))
     if protos:
-      print "Attempting to build proto files:\n%s" % '\n'.join(protos)
-      subprocess.check_call([
+      print 'Attempting to build proto files:\n%s' % '\n'.join(protos)
+      cmd = [
           self.protoc,
           '--proto_path', self.indir,
           '--proto_path', self.protodir,
           '--python_out', self.outdir,
-          ] + protos)
+      ] + protos
+      try:
+        subprocess.check_call(cmd)
+      except OSError as e:
+        if e.errno == errno.ENOENT:
+          print 'Could not find the protobuf compiler at %s' % self.protoc
+          print ('On many Linux systems, this is fixed by installing the '
+                 '"protobuf-compiler" and "libprotobuf-dev" packages.')
+        raise
+      except subprocess.CalledProcessError:
+        print 'Could not build proto files.'
+        print ('This could be due to missing helper files. On many Linux '
+               'systems, this is fixed by installing the '
+               '"libprotobuf-dev" package.')
```

Comment:
oh, uh, this should raise too... otherwise it silently keeps going and it looks like the build succeeded.